src/backend/commands/user.c

```diff
@@ -554,31 +541,34 @@ CreateRole(CreateRoleStmt *stmt)
 
 	if (resqueue)
 	{
-		if (strcmp(resqueue, "none") == 0) {
+		if (strcmp(resqueue, "none") == 0)
+		{
 			unregisterConnectionInRMWithErrorReport(resourceid);
 			releaseResourceContextWithErrorReport(resourceid);
 			ereport(ERROR,
 					(errcode(ERRCODE_RESERVED_NAME),
-					 errmsg("resource queue name \"%s\" is reserved",
+					 errmsg("Resource queue name \"%s\" is reserved",
```

Comment:
According to the PostgreSQL Error Message Style Guide, "Do not capitalize the first letter". I think here the old one is better than the new one.